### ISSUE TITLE
[DOCS] Make getting started more inclusive

### ISSUE THREAD

**Issue by goobric** [EddieHubCommunity/LinkFree#1221]
### Description

The Prerequisites; Installation Steps; and To Add Your Profile
for a person who is truly 'new2tech' (and not necessarily a 'code-newbie') this is very daunting and a completely new language.
Not inclusive for a person who is 'new2tech'

### Screenshots

![220403_eddieHubDoc](https://user-images.githubusercontent.com/23141638/161434806-1d88b3c4-ff1c-4c9d-b295-16cd9ca27fd4.png)

### Additional information

Sign-posts to simple explanation resources (whatever type of media that maybe) that would help the person who is new2tech to be involved.
After the person has reviewed the signposts, they can then ask for clarity if need be; and then choose to contribute.
The Installation Steps rely on someone that has done this already and has experience of already doing this. For a person new to GitHub (and say again new2tech) they will not know how to do this; unless there is a working-example of how to do this.

**Comment by emmalearnscode** [comment]
I wonder if a link to a YouTube video showing these steps would be a good idea.

**Comment by goobric** [comment]
most definitely 👍
sign posting to resources that explain these concepts in a simple way (so that a 10yr old could understand) would be very beneficial.

**Comment by KarlGusta** [comment]
I would also suggest we have a Wiki that can have links to the various get started tutorials. Also a documentation part on the website where we can edit from GitHub? 

I would like to work on this if the maintainers would allow.

**Comment by goobric** [comment]
I have created a basic file with links to resources to assist with downloading LinkFree
please amend if anything is incorrect
https://github.com/goobric/linkfreeInformation

**Comment by emmalearnscode** [comment]
> I have created a basic file with links to resources to assist with downloading LinkFree please amend if anything is incorrect https://github.com/goobric/linkfreeInformation

This looks like a great start. I wonder if we should have it as a general document within the organisation that can be linked to  from all repos (I don't see anything super specific to LinkFree although I only looked at it briefly so I might've missed something?) and I wonder if some of Eddie's github videos should also be included as reference material.

**Comment by schmelto** [comment]
Agree with Emma

**Comment by emmalearnscode** [comment]
That’s a great point @goobric I just read through and it would definitely make sense to move that warning

**Comment by goobric** [comment]
please excuse my naivety, who has the responsibility to amend/update the instructions for the LinkFree project?
Can I submit a PR with proposed alterations?

**Comment by emmalearnscode** [comment]
> please excuse my naivety, who has the responsibility to amend/update the instructions for the LinkFree project?
> Can I submit a PR with proposed alterations?

Yes, you can make a PR. No one person/group is responsible for the instructions, everyone can contribute to improve them.

**Comment by marianaWest** [comment]
Hello, first-time contributor here! 
I came into this issue because I also had a problem with the "To Add Your Profile" session. I agree with @goobric that the instructions are not clear to someone new2tech (like me). 
I would suggest something similar to what he said: 
"In the data directory, create a .json file with the same name as your GitHub username. For example, if your GitHub username is eddiehub, create a file named eddiehub.json. This will ensure that the name is unique."
I also want to point out that maybe we can avoid using abbreviations (like "i.e.") as they are not necessarily friendly to people whose first language is not English. 
I saw that @goobric added a commit, but I saw that the changes haven't been incorporated yet, probably because maintainers are busy. In any case, I hope this helps, and sorry for the long comment!
